sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/README.md

```diff
@@ -135,6 +133,42 @@ For more information on the OpenTelemetry project, please review the [OpenTeleme
 
 Refer to [`Program.cs`](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/tests/Azure.Monitor.OpenTelemetry.Exporter.Demo/Program.cs) for a complete demo.
 
+### Log Scopes
+
+Log scopes allow you to add additional properties to the logs generated by your application.
+Although the Azure Monitor Exporter does support scopes, this feature is off by default in OpenTelemetry.
+To leverage log scopes, you must explicitly enable them.
+
+To include the scope with your logs, set `OpenTelemetryLoggerOptions.IncludeScopes` to `true` in your application's configuration:
+```csharp
+var loggerFactory = LoggerFactory.Create(builder =>
+{
+    builder.AddOpenTelemetry(options =>
+    {
+        options.AddAzureMonitorLogExporter(o => o.ConnectionString = "InstrumentationKey=00000000-0000-0000-0000-000000000000");
+        options.IncludeScopes = true;
+    });
+});
+```
+
+Once enabled, you can define a scope using a dictionary or any enumerable of `KeyValuePair<string, object>`.
```

Comment:
When using `ILogger`
  [scopes](https://learn.microsoft.com/dotnet/core/extensions/logging#log-scopes),
  use a `List<KeyValuePair<string, object?>>` or `IReadOnlyList<KeyValue<string,
  object?>>` as the state for best performance. Use an
  `IEnumerable<KeyValue<string, object?>>` when performance is *NOT* a critical
  requirement.

use the above wording. I had it from a previous discussion. It should ideally be in OTel upstream, but OTel has been shying away from Log Scopes... so lets keep it here atleast.

Reply:
> Use an IEnumerable<KeyValue<string, object?>> when performance is NOT a critical requirement.

This seems weird to me. I'm thinking we shouldn't include non-recommended approaches.
Also, getting deep into the performance details of ILogger seems off-topic for Azure Monitor. 
That seems more appropriate for ILogger's own docs